plugin/trino-kudu/src/main/java/io/trino/plugin/kudu/KuduPageSink.java

```diff
@@ -125,7 +128,10 @@ private KuduPageSink(
             }
 
             try {
-                session.apply(upsert);
+                OperationResponse response = session.apply(upsert);
```

Comment:
Can we extract it from the version update ? It looks like `OperationResponse` is returned in old APIs too.

Reply:
Same comment as above, there is no way to easily verify the fix because you need a very specific sequence of timeouts [no timeouts when initially connecting to kudu, but timeouts during deletes/upserts/etc]. The kudu 1.15.0 client [accidentally] provides those timeouts due to a bug in the client.

See here for the failing kudu tests when upgrading to kudu 1.15.0:
https://github.com/trinodb/trino/runs/5057440792?check_suite_focus=true

And here for the workaround that makes tests pass:
https://github.com/trinodb/trino/runs/5060298096?check_suite_focus=true

If we do not mind having no tests around the change, I can extract this into a separate PR.